automerge-backend-wasm/scripts/js_tests.sh

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env bash
+
+if [[ -e "../../automerge" ]]
+then
+    echo "Automerge repo exists. Skipping cloning..."
+else
+    echo "Cloning automerge repo to ../../automerge"
+    git clone https://github.com/automerge/automerge.git ../../automerge
+fi
+
+ORIGINAL_PWD=$PWD
+cd ../../automerge
+git checkout 81079ff75d2234b47cb912bad728158f2e71c527
```

Comment:
Should there be some sort of check to see if this commit even exists? (could have an old repo so could ask user to update)

Reply:
Done